<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.core.NestedExceptionTests" tests="4" skipped="0" failures="0" errors="0" timestamp="2019-07-23T02:43:49" hostname="zhoujiajundeMacBook-Pro.local" time="0.008">
  <properties/>
  <testcase name="nestedRuntimeExceptionWithRootCause" classname="org.springframework.core.NestedExceptionTests" time="0.003"/>
  <testcase name="nestedCheckedExceptionWithRootCause" classname="org.springframework.core.NestedExceptionTests" time="0.001"/>
  <testcase name="nestedRuntimeExceptionWithNoRootCause" classname="org.springframework.core.NestedExceptionTests" time="0.001"/>
  <testcase name="nestedCheckedExceptionWithNoRootCause" classname="org.springframework.core.NestedExceptionTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
